### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jansvensson**
Added some troubleshooting tips and removed the link to the original dcs bios as it does not seem to work.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,11 +19,11 @@ TouchDCS is an interface between OSC applications (in this case, TouchOSC) and D
 
 ## Getting Started
 ### What you'll need:
-1. DCS-BIOS ([Original](https://github.com/dcs-bios/dcs-bios) or [Flightpanels fork](https://github.com/DCSFlightpanels/dcs-bios))
-2. [TouchOSC](https://hexler.net/products/touchosc) on your phone/tablet
+1. DCS-BIOS [Flightpanels fork](https://github.com/DCSFlightpanels/dcs-bios))
+3. [TouchOSC](https://hexler.net/products/touchosc) on your phone/tablet
```
Comment (charliefoxtwo) [inline README.md:23]: Any reason these numbers changed? It doesn't really matter since it will render the same I guess, meh.

**Review by charliefoxtwo — APPROVED** [review]
Generally lgtm, just the one nit. You're right that a better explanation is needed, but perfect is the enemy of good and these changes are still far better than the nothing that exists right now.

Fwiw I'll probably end up moving this stuff to the wiki this weekend, so fear not if it disappears.

File: README.md (hunk 1 above)
Comment (jansvensson) [reply README.md]: Fixed in a new commit

Comment (jansvensson) [reply README.md]: Btw, its the first time I am collaborating over the internet in an open source project, so I am sorry if I am causing any troubles.

Comment (charliefoxtwo) [reply README.md]: No, not at all! Everybody has to start somewhere and it's not the most self-explanatory process. I appreciate your contribution :)

**Comment by charliefoxtwo** [comment]
Thanks!

**Pull request merged**